### ISSUE TITLE
BaseFragment should be renamed

### ISSUE THREAD

**Issue by dri94** [Instabug/Instabug-Android#188]
Instabug is using BaseFragment as its custom fragment name. However this is a common name used by most companies to remediate the issue of duplicate code. It causes confusion when using the IDE and having multiple "BaseFragment"'s pop up. Instabug should rename their's to InstabugFragment to make it clear and not cause confusion for other's in their projects.

### Steps to Reproduce the Problem

Create a base fragment class for project such as
```
abstract class BaseFragment<T : Fragment>: Fragment() {
}
```
Try to extend another fragment in project from recently created BaseFragment. 
Notice instabug's BaseFragment class pops up as an option, which can easily be accidently pressed.
### Expected Behavior
Instabug should define their custom fragment as "InstabugFragment" or something along those lines
### Actual Behavior
SDK currently uses a name that causes conflicts and is easily confused when working in an enterprise project

### Instabug integration code
Just add to build.gradle

### SDK Version
8.2.2
### Android Version
All
### Device Model
All

**Comment by hanaamohamed** [comment]
Hello @dri94, You can use `com.instabug.library:instabug:8.4.0.2080-SNAPSHOT` this snapshot includes the renaming, also in order to consume the snapshot you will need to add the below snippet 
```
 repositories {

        maven {
            url "https://oss.sonatype.org/content/repositories/snapshots"
        }

    }

```

**Comment by mzelzoghbi** [comment]
Hi @dri94, we included this change in our latest release **8.5.0**, please get back to us if you encounter any other issue. Thank You 😃
